deny.toml

```diff
@@ -0,0 +1,144 @@
+# See all available configuration options at:
+# https://embarkstudios.github.io/cargo-deny/index.html
+
+# --------------------------------------------------
+# General
+# --------------------------------------------------
+
+# If true, metadata will be collected with `--all-features`
+all-features = true
+# If true, metadata will be collected with `--no-default-features`
+no-default-features = true
```

Comment:
Is `--no-default-features` necessary if we are going to enable `--all-features` anyway?

Reply:
The way I read the documentation was:
- `all-features = `true` ensures we check for dependencies with all of the features.
- `no-default-features = true` will also check the library with no default features enables.

Happy to change this to `false` if you think that's correct @rushmorem?

Reply:
Ah I see! I have never used `cargo deny` before. I was just going off of those configuration options. I thought `all-features` simply enables `--all-features` and `no-default-features` enables `--no-default-features` to end up with

```
--no-default-features --all-features
```

That's fine with me then! Thanks for the explanation @tobiemh!